### PR TITLE
New feature: Alerts List subscriptions/preference settings

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Review by MariePMcGarry — APPROVED** [review]
lgtm

File: docs/alerts/monitors/alert-response.md [hunk 1]
```diff
@@ -41,11 +43,17 @@ If you use [Webhook connections](/docs/manage/connections-integrations/webhook-c
...
 Alert List shows all the Alerts from triggered Monitors within the past seven days. By default, the list is sorted by status (showing **Active** on top, followed by **Resolved**), and then chronologically by creation time.
 
-To open the Alert List, click the bell icon in the top menu. <br/>![alert list page with bell and border.png](/img/monitors/alert-list-page-bell-border.png)
-* Use the search bar to filter by name, severity, or status.<br/>![search alert list.png](/img/monitors/search-alert-list.png)
-* To sort the list by category (e.g., Severity, Name, ID), click on a column header.
-* To resolve an alert, click a row to select it and use the **Resolve** button.
-* To open an alert in the Alerts page to further investigate, double-click on a row.
+To open the Alert List, click the bell icon in the top menu. <br/> <img src={useBaseUrl('img/alerts/alert-list-page-bell-border.png')} alt="alert-list-page-bell-border" width="300"/>
+
+To filter or sort by category (e.g., **Name**, **Severity**, **Status**), you can you the search bar or click on a column header.<br/>![search alert list.png](/img/monitors/search-alert-list.png)
```
[the remark this answers is not in the document]
Comment (kimsauce) [reply docs/alerts/monitors/alert-response.md:48]: good catch!

File: docs/alerts/monitors/alert-response.md [hunk 2]
```diff
@@ -162,6 +170,23 @@ For example, the card below shows that “ServiceUnavailable” error is happeni
 * **E** - expand / collapse details panel.
 * **F** - opens a Log Search filtered to the Log messages that match the dimensional details of the telemetry value
 
+## Subscribe to Alerts
```
[the remark this answers is not in the document]
Comment (kimsauce) [reply docs/alerts/monitors/alert-response.md:173]: @dkarabin-sumo good thinking. Can you help me fill in the blanks below?

There are several places where you can subscribe to an alert:
* From your Alert list, hover your mouse over a row, click the three-dot kebab menu, and select **Subscribe**.
* From the Alert page, ?
* From the Monitors List page, ?

To subscribe to a monitor, go to your Alert list, click on a row item, then click **Subscribe to Monitor**.

File: docs/alerts/monitors/alert-response.md [hunk 3]
```diff
@@ -162,6 +170,23 @@ For example, the card below shows that “ServiceUnavailable” error is happeni
...
+
+To subscribe to an alert, go to your Alert List, hover your mouse over a row, click the three-dot kebab menu, and select **Subscribe**.
```
[the remark this answers is not in the document]
Comment (kimsauce) [reply docs/alerts/monitors/alert-response.md]: thank you for the clarification! this helps.

File: docs/alerts/monitors/alert-response.md [hunk 4]
```diff
@@ -162,9 +169,30 @@ For example, the card below shows that “ServiceUnavailable” error is happeni
...
+## Subscribe to Alert Monitors
+
+:::note Terminology
+A _Monitor_ creates an _Alert_. You can subscribe to an _Alert_'s _Monitor_.
+:::
+
+There are several places where you can subscribe:
+* From your Alert list, hover your mouse over a row, click the three-dot kebab menu, and select **Subscribe**.
+* From your Alert list, click on a row item, then in the side panel (Alert page), click **Subscribe to Monitor**.
+* ?
```
Comment (kimsauce) [inline docs/alerts/monitors/alert-response.md:181]: @dkarabin-sumo , can I get your help finalizing this section?

File: docs/alerts/monitors/alert-response.md [hunk 5]
```diff
@@ -162,9 +169,37 @@ For example, the card below shows that “ServiceUnavailable” error is happeni
...
+
+
+#### From your Alerts list
+* Right-click on a row item > click **Subscribe**
+* Hover your mouse over a row, click the three-dot kebab menu > select **Subscribe**
+* Single-click on a row item > in the side panel (Alert page), click the three-dot kebab menu > **Subscribe to Monitor**
+
+
+#### From your Monitors list
+* Right-click on a row item > click **Subscribe**
```
Comment (kimsauce) [inline docs/alerts/monitors/alert-response.md:186]:
```suggestion
* Right-click on a row item > click **hello**
```

**Pull request merged**